LongestPalindrome.java

```diff
@@ -0,0 +1,31 @@
+// ## Problem3 (https://leetcode.com/problems/longest-palindrome)
+
+// Time Complexity : O(1)
```

Comment:
Kindly check time and space complexities. TC : O(N) , SC:O(1) as it just contains upper and lower case characters.